packages/firestore-types/index.d.ts

```diff
@@ -57,6 +57,22 @@ export interface Settings {
   timestampsInSnapshots?: boolean;
 }
 
+/** Settings used to configure Firestore persistence. */
```

Comment:
To make life easier for people that are looking at docs / not using TypeScript, I'd say "Settings that can be passed to Firestore.enablePersistence() to configure ..." or something

Reply:
Done